<?php

throw new Exception('test exception 2');
